src/Microsoft.DocAsCode.MarkdownLite/JsonTokenTreeRenderer.cs

```diff
@@ -0,0 +1,284 @@
+        {
+            return this.Insert("Escape" + token.Content, this.GetSize(), Type.Leaf);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownLinkInlineToken token, MarkdownInlineContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Content)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Link", childContent, Type.NonLeaf);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownImageInlineToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("Image", this.GetSize(), Type.Leaf);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownStrongInlineToken token, MarkdownInlineContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Content)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Strong", childContent, Type.NonLeaf);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownEmInlineToken token, MarkdownInlineContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Content)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Em", childContent, Type.NonLeaf);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownCodeInlineToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("InLineCode", this.GetSize(), Type.Leaf);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, GfmDelInlineToken token, MarkdownInlineContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Content)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Del", childContent, Type.NonLeaf);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownTagInlineToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("Tag", this.GetSize(), Type.Leaf);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownBrInlineToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("Br", this.GetSize(), Type.Leaf);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownTextToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("InLineText(" + token.Content.Replace("\n", "\\n") + ")", this.GetSize(), Type.Leaf);
```

Comment:
>Replace [](start = 61, length = 7)

What about .Replace("\", "\\")? some other case may break this, like: "\r", "aa\".
Also consider replace `"` with `\"`?